web-app/js/portal/details/NcWmsPanel.js

```diff
@@ -323,7 +323,6 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
         if (this.geoNetworkRecord) {
             this._addDateTimeFilterToLayer();
             this.geoNetworkRecord.updateNcwmsParams(dateRangeStart, dateRangeEnd, geometry);
-            Ext.MsgBus.publish(PORTAL_EVENTS.DATA_COLLECTION_MODIFIED);
```

Comment:
This is now a while ago. It was the cause of an infinite loop in my changes. As to why it was needed in the first place ... no idea.
Basically now the ActiveGeoNetworkRecordStore triggers this event
the DownloadPanel and SubbsettingPanel listen
So the event is more like AN_ITEM_IN_THE_DATA_COLLECTION_STORE_IS_MODIFIED_OR_JUST_THE_ORDER